locales/native-names.json

```diff
@@ -8,6 +8,7 @@
   "as": "অসমীয়া",
   "ast": "Asturianu",
   "az": "Azərbaycanca",
+  "be": "be",
```

Comment:
looks like there is no be translation yet, and therefore there is no translated native name